### PR TITLE
OO#86 welcome messages can render HTML

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

File: src/main/scala/ch/openolitor/core/db/evolution/scripts/v2/OO86_EnlargeLoginMessage.scala [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import ch.openolitor.stammdaten.StammdatenDBMappings
+import com.typesafe.scalalogging.LazyLogging
+import scalikejdbc._
+
+import scala.util.{ Success, Try }
+
+object OO86_EnlargeLoginMessage {
+  val EnlargeLoginMessage = new Script with LazyLogging with StammdatenDBMappings {
+    def execute(sysConfig: SystemConfig)(implicit session: DBSession): Try[Boolean] = {
+      sql"""ALTER TABLE ${projektMapping.table} MODIFY welcome_Message1 MEDIUMTEXT""".execute.apply()
```
Comment (rawyler) [inline src/main/scala/ch/openolitor/core/db/evolution/scripts/v2/OO86_EnlargeLoginMessage.scala]: 16MB of data, nice ;) didn't know that

File: src/main/scala/ch/openolitor/core/db/evolution/scripts/v2/V2Scripts.scala [hunk 2]
```diff
@@ -249,5 +249,6 @@ object V2Scripts {
     OO_gitlab_373_ZahlungsEingang_teilnehmer_nummer.scripts ++
     OO_distribution_date_time_should_not_consider_time.scripts ++
     Operations465_EnlargeKundeBez.scripts ++
-    OO411_adding_person_contact_permission.scripts
+    OO411_adding_person_contact_permission.scripts ++
+    OO86_EnlargeLoginMessage.scripts
```
Comment (rawyler) [inline src/main/scala/ch/openolitor/core/db/evolution/scripts/v2/V2Scripts.scala]: The naming pattern of the scripts seems to have diverged over time. I suggest deciding on one style for the future or even rename `OO86_EnlargeLoginMessage` to `OO86_increase_login_message_size`

File: src/main/scala/ch/openolitor/core/db/evolution/scripts/v2/OO86_EnlargeLoginMessage.scala (hunk 1 above)
Comment (mcmpp) [reply src/main/scala/ch/openolitor/core/db/evolution/scripts/v2/OO86_EnlargeLoginMessage.scala]: fancy

File: src/main/scala/ch/openolitor/core/db/evolution/scripts/v2/V2Scripts.scala (hunk 2 above)
Comment (mcmpp) [reply src/main/scala/ch/openolitor/core/db/evolution/scripts/v2/V2Scripts.scala]: done

**Review by rawyler — APPROVED** [review]

**Pull request merged**